### PR TITLE
Add Write to Delta Lake from Flink blog

### PR DESCRIPTION
**Pull request by dennyglee**
Adding the "Write to Delta Lake from Apache Flink" blog

### CODE REVIEW

File: src/blog/2022-04-27-writing-to-delta-lake-from-apache-flink/index.mdx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+### DeltaWriter
+
+The goal of a <code>DeltaWriter</code> is to manage bucket writers for partitioned tables and pass incoming events to the correct bucket writer. One writer can write data to multiple buckets (also called partitions) at the same time but only one file per bucket can be in progress (aka “open”) state. If the Delta table is not partitioned, then there will be only one bucket writer for one DeltaWriter that will be writing to the table’s root path.
+
+Delta files can be in 3 different states:
+
+- **In-progress**: Files to which writers are currently appending new records, they are also called “opened” files.
+- **Pending** : Files that have already been closed and are waiting to be committed to the Delta log during the next checkpoint. These files are in their final state and will not be appended anymore, only renamed during <code>DeltaCommitter.commit</code> operation.
+- **Finished** : Files already written, closed, and successfully committed to Delta table
```
Comment (scottsand-db) [inline src/blog/2022-04-27-writing-to-delta-lake-from-apache-flink/index.mdx:49]: nit: end in period `.`

File: src/blog/2022-04-27-writing-to-delta-lake-from-apache-flink/index.mdx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+### DeltaGlobalCommitter
+
+The Global Committer combines multiple lists of DeltaCommittables received from multiple DeltaCommitters and commits all files to the Delta log.
+
+## Architecture
+
+The following architecture diagram illustrates how the data is written from a Flink application to Delta Lake tables. Each Flink job can have multiple parallel <code>DeltaWriters</code>, <code>DeltaCommitters</code>, and only one <code>DeltaGlobalCommitter</code>. For each checkpoint, DeltaWriter combines a list of DeltaCommittables from multiple bucket writers and sends it to the DeltaCommitter instance, which then is responsible for locally committing the files and marking them ready to be committed to the Delta log. DeltaGlobalCommiter combines the DeltaCommitables from all the DeltaCommitters and commits the files to the Delta Log.
```
Comment (scottsand-db) [inline src/blog/2022-04-27-writing-to-delta-lake-from-apache-flink/index.mdx:65]: plurality
```
multiple parallel <code>DeltaWriter</code>s, <code>DeltaCommitters</code>s
```

**Review by scottsand-db — APPROVED** [review]
2 minor nots, otherwise LGTM!

File: src/blog/2022-04-27-writing-to-delta-lake-from-apache-flink/index.mdx (hunk 2 above)
Comment (dennyglee) [reply src/blog/2022-04-27-writing-to-delta-lake-from-apache-flink/index.mdx]: Fixed - thanks!

File: src/blog/2022-04-27-writing-to-delta-lake-from-apache-flink/index.mdx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+## Architecture
+
+The following architecture diagram illustrates how the data is written from a Flink application to Delta Lake tables. Each Flink job can have multiple parallel <code>DeltaWriter</code>s, <code>DeltaCommitter</code>s, and only one <code>DeltaGlobalCommitter</code>. For each checkpoint, DeltaWriter combines a list of DeltaCommittables from multiple bucket writers and sends it to the DeltaCommitter instance, which then is responsible for locally committing the files and marking them ready to be committed to the Delta log. DeltaGlobalCommiter combines the DeltaCommitables from all the DeltaCommitters and commits the files to the Delta Log.
+
+![Apache Flink App with Delta Lake](./apache-flink-app-with-delta-lake.png)
+
+### Implementation - Writing data from Flink into Delta Lake tables
+
+The [Flink/Delta Lake Connector](https://github.com/delta-io/connectors/tree/master/flink-connector/) is a JVM library to read and write data from Apache Flink applications to Delta Lake tables utilizing the [Delta Standalone JVM library](https://github.com/delta-io/connectors#delta-standalone). It includes:
```
Comment (scottsand-db) [inline src/blog/2022-04-27-writing-to-delta-lake-from-apache-flink/index.mdx:71]: `The [Flink/Delta Lake Connector](https://github.com/delta-io/connectors/tree/master/flink/)`

not `flink-connector`

**Pull request merged**